lsm-perf.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env
+
+import argparse
+import time
+import plumbum
+import sys
+import statistics
+from plumbum import local, SshMachine  # TODO: requirements
+
+
+ROUNDS = 3
+REPETITIONS_PER_ROUND = 50
+WARMUP_RUNS = 5
+ON_VM_WORKLOAD_PATH = '~/lsm-perf-workload'
+QEMU_EXIT_CMD = b'\x01cq\n'  # -> ctrl+a c q
+
+
+def main(args):
+    try:
+        init_output_file(args.out)
+        for round in range(ROUNDS):
+            print('Starting round %d' % round)
+            for kernel in args.kernels:
+                results = evaluating_kernel(
+                    kernel_path=kernel.name,
+                    img_path=args.image.name,
```

Comment:
I think we'd better call it `filesystem_img` or something - `img` is a bit vague

Reply:
Sounds good, I'll change it

Reply:
I think `filesystem_path` is also a bit confusing - if you don't know exactly what it's referring to it might make you think "all paths are filesystem paths" - maybe `filesystem_img_path`?

Reply:
Yes, I misread your original comment. I changed it too `filesystem_img_path` now.